botbuilder-teams-js/src/activityProcessor/teamsFactory.ts

```diff
@@ -162,4 +164,9 @@ export class TeamsFactory extends CardFactory {
       toAdaptiveCardAction
     };
   }
+
+  public static stripHtmlTag(content: string): string {
+    const dom = new JSDOM(content);
+    return dom.window.document.body.textContent;
```

Comment:
As we discussed yesterday, let's use DOM traversing algorithm to ensure producing the same results for both C# & Node.js.

Reply:
done